libraries/botbuilder-core/src/cardFactory.ts

```diff
@@ -156,6 +156,15 @@ export class CardFactory {
         buttons?: (CardAction | string)[],
         other?: Partial<HeroCard>
     ): Attachment;
+    /**
+     * Returns an attachment for a hero card.
+     *
```

Comment:
This space between the summary and the params is not present in the rest of the comments

Reply:
It's like this in the previous overloads of this method, that's why I kept it.